packages/devtools_app/lib/src/app.dart

```diff
@@ -613,15 +625,10 @@ List<DevToolsScreen> get defaultScreens {
       VMDeveloperToolsScreen(controller: vmDeveloperToolsController),
       controller: vmDeveloperToolsController,
     ),
-    DevToolsScreen<void>(
```

Comment:
merge conflict. Looks like you accidentally removed the provider screen.

Reply:
Good catch I didn't notice that. Fixed.